back/engines/commercial/custom_maps/app/services/custom_maps/side_fx_map_config_service.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+module CustomMaps
+  class SideFxMapConfigService < BaseSideFxService
+    include SideFxHelper
+
+    def before_create(_map_config, _user); end
+
+    def before_update(_map_config, _user); end
+
+    def before_destroy(_map_config, _user); end
+
+    def after_destroy(frozen_resource, user)
+      serialized_resource = clean_time_attributes(frozen_resource.attributes)
+      clean_rgeo_attributes(serialized_resource)
+
+      LogActivityJob.perform_later(
+        encode_frozen_resource(frozen_resource),
+        'deleted', user, Time.now.to_i,
+        payload: { resource_name => serialized_resource }
+      )
+    end
```

Comment:
Overrides the `BaseSideFxService` method in order to make use of the new `clean_rgeo_attributes` method.

Reply:
Is `clean_rgeo_attributes` used anywhere else? If not, then not sure why it's not defined in this service

Reply:
True. I'll move it into the service.